src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -241,16 +240,17 @@ def get_parser() -> argparse.ArgumentParser:
         "--all, -all, -a", action="store_true", dest="all", help="List all folders"
     )
     parser.add_argument(
-        "--verbosity, -v",
+        "--verbose, --verbosity, -v",
         action="store_true",
         dest="verbosity",
         help="Enable logging (messages) for debugging",
     )
     parser.add_argument(
         "--cleanup",
         action="store_true",
```

Comment:
This probably does not work if the default is True.

Has this been True by default in the legacy version? Otherwise I would prefer this to be False by default and error hard if target exists.